src/implementations/twig/components/button/button.html.twig

```diff
@@ -97,8 +98,16 @@
       {%- if icon.extra_classes is defined and icon.extra_classes is not empty -%}
         {% set _icon_extra_classes = _icon_extra_classes ~ ' ' ~ icon.extra_classes %}
       {%- endif -%}
+      {% set _icon_extra_accessibility = {} %}
```

Comment:
mmhh..isn't it better if we do this in the icon template? I mean that the user would normally be able to set this title passing it as part of the extra_accessibility, we do what is in this pr i guess in order to "simplify" the data structure and let the user pass a "title" prop in the main icon object, instead. But then we have to do this for all the components including the icon, basically, why this could be an option available at the icon template level, so that we could check for this "simplified" title and use it in case it is present while still relying normally on the extra_accessibility object for that.